docs/sql-guide/statements/statement-view-create.md

```diff
@@ -0,0 +1,107 @@
+---
+title: CREATE VIEW
+layout: default
+parent: Statements
+grand_parent: SQL guide
+nav_order: 2
+---
+
+## CREATE VIEW statement
+
+Creates a FeatureBase view. The view already exists and `IF NOT EXISTS` is not specified the statement will not be successful.
+
+## BNF diagrams
+
+![expr](/assets/images/sql-guide/create_view_stmt.svg)
+
+## DDL Syntax
+
+```
+CREATE VIEW view_name
+  [IF NOT EXISTS]
```

Comment:
Is this in the correct order? doesn't this come before "view_name"?

Reply:
tested, yes. there is a same error on `create table` as well. updated